install_scripts/ubuntu_install.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/sh
+
+repository="palmtop/wsdd"
+
+#download wsdd python script
+rm -f wsdd.py
+wget -q https://github.com/$repository/raw/master/src/wsdd.py
```

Comment:
Referencing to `master` might not be a good idea, if master is broken. As long as possibly bleeding edge is wanted then master is ok 😉 Referencing a release is a better approach, see [the Gentoo packages](/christgau/wsdd-gentoo/tree/master/net-misc/wsdd).